packages/notifi-react-card/lib/components/intercom/ChatMessageSection.tsx

```diff
@@ -47,8 +53,6 @@ export const ChatMessageSection: React.FC<ChatMessageSectionProps> = ({
         />
       ) : (
         <Virtuoso
-          //TODO: improvement to add the scrollSeekConfiguration property
```

Comment:
Can we please still add this to-do? I think that it is still a good to-do :)

Reply:
Ah, delete the to-do accidentally, added